app/assets/javascripts/student_profile_v2/profile_details.js

```diff
@@ -3,14 +3,46 @@
   var dom = window.shared.ReactHelpers.dom;
   var createEl = window.shared.ReactHelpers.createEl;
   var merge = window.shared.ReactHelpers.merge;
-
+  var FeedHelpers = window.shared.FeedHelpers;
   var QuadConverter = window.shared.QuadConverter;
 
+  var Colors = {
+    math: '#e8e9fc',
+    reading: '#ffe7d6',
+    procedural: '#e8fce8'
+  };
+  var styles = {
+    feedCard: {
+      marginBottom: '10px',
+      borderRadius: '10px',
+      fontSize: '10pt'
+    },
+    feedCardHeader: {
+      fontSize: '120%',
```

Comment:
Ahh I see, I missed the `80%` inline.  Yeah I think my general suggestion would be: if there are font size changes, let's see if we can express them in pixels (with percentages we lose isolation, in that one change can cascade and in general it'd be better to avoid that).  And if there are places where there are many font size changes, then it's probably worth looking at simplifying to a fewer number of font sizes.

So here, would removing the sizing on the container simplify?  Then each element is sized individually and with pixels, and they're each isolated and not interdependent?  Let me know if I'm being confusing here. :)